app/components/sdg/related_list_selector_component.html.erb

```diff
@@ -12,9 +12,16 @@
     <%= f.text_field :related_sdg_list,
                       class: "input",
                       placeholder: t("sdg.related_list_selector.placeholder"),
-                      label: t("sdg.related_list_selector.hint"),
+                      label:  t("sdg.related_list_selector.label"),
```

Comment:
Maybe we can move this translation to `activerecord.yml` (again :sweat_smile:), like it was before [commit ab2b3a267](https://github.com/consul/consul/commit/ab2b3a26716d4ce5987310205dd2b5e1f1290912#diff-0f1a8cdb0736d2fd3a4b2e34d155932946101748715cd2b5b1b5cb5c1dc39488L5) :thinking:. That way we can omit this line.